go.mod

```diff
@@ -27,7 +27,7 @@ require (
 	filippo.io/edwards25519 v1.0.0-beta.2 // indirect
 	github.com/99designs/keyring v1.1.6 // indirect
 	github.com/ChainSafe/go-schnorrkel v0.0.0-20200405005733-88cbf1b4c40d // indirect
-	github.com/CosmWasm/wasmvm v1.0.0-soon2.0.20211209162446-b96c92d21aaf // indirect
```

Comment:
This is the problem. @faddat, can upgrade the Notional Labs fork to use `v1.0.0-beta3`? It's the latest release. https://github.com/CosmWasm/wasmvm/releases

7 is a lucky number.

Reply:
so, we're using beta2 currently, but golang tags stuff kinda odd.

Here's the pr for beta3:
[
](https://github.com/CosmosContracts/juno/pull/127)